go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/golang-jwt/jwt/v5
 
-go 1.16
+go 1.20
```

Comment:
Does this need to change to 1.20, or can this remain on the minimum version we currently support?

Reply:
I thought I reverted this back to Go 1.18 at least, I ll double check, maybe I only edited the CI file. I think per our 2-release-support strategy we can bump this to 1.18.